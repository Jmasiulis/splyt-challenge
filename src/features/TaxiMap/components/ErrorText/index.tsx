import React from 'react';

import StyledErrorText from './styles';

export default function ErrorText(): React.ReactElement {
  return (
    <StyledErrorText>
      Error has occurred while fetching the data!
    </StyledErrorText>
  );
}
