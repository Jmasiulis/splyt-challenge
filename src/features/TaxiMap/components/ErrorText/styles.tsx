import styled from 'styled-components';

const StyledErrorText = styled.p`
  color: red;
  margin: 10px;
`;

export default StyledErrorText;
