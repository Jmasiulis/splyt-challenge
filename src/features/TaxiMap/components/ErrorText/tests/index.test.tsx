import React from 'react';
import { shallow } from 'enzyme';

import ErrorText from '..';

describe('<ErrorText />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(<ErrorText />);
    expect(wrapper).toMatchSnapshot();
  });
});
