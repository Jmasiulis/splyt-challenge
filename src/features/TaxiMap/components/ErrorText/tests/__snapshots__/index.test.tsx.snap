// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorText /> should render with default props 1`] = `
<styled.p>
  Error has occurred while fetching the data!
</styled.p>
`;
